CONTRIBUTING.md

```diff
@@ -10,32 +10,78 @@ You can do this online, and it only takes a minute. If you've never submitted co
 you must add your (or your organization's) name and contact info to the [AUTHORS](AUTHORS)
 file.
 
-## Development (Pure web)
+## Development
 
 - `git clone https://github.com/flutter/devtools`
 - `cd devtools/packages/devtools_app`
-- `pub get`
+- `flutter pub get`
 
 From a separate terminal:
 - `cd <path/to/flutter-sdk>/examples/flutter_gallery`
 - ensure the iOS Simulator is open (or a physical device is connected)
 - `flutter run`
 
+### Pure-Dart Web App
```

Comment:
nit: call this dart:html Web App or "Legacy dart:html Web App"

Reply:
Done.